### DISCUSSION TITLE
Fermato Auto V ip TV

### DISCUSSION THREAD

**Discussion by asterix32** [AndreyPavlenko/Fermata#144] [General]
I'm trying to enter my IP TV subscription from the TV viewing section of Fermato Auto, but I couldn't activate it. Is it possible to explain with a picture, I wonder I have a subscription to IP TV

**Comment by AndreyPavlenko** [comment]
Do you have a link to m3u playlist? If yes, press the add button in the bottom right corner and add the playlist url.

**Reply by asterix32** [reply]
Yes, I add it, but I don't know where to enter my username and password. Only when I add the url, the channels do not come.

**Comment by AndreyPavlenko** [comment]
There are not such options for the username and password. You need to have the direct URL. Usually, the username and password are specified in the URL.

**Reply by asterix32** [reply]
After the url, I uploaded the url with the username and password extension and it worked, thank you. I would like to be in direct contact with you
